client/my-sites/email/paths.ts

```diff
@@ -0,0 +1,189 @@
+import { stringify } from 'qs';
+import { isUnderDomainManagementAll, domainManagementRoot } from 'calypso/my-sites/domains/paths';
+
+type QueryStringParameters = { [ key: string ]: string | undefined };
+
+type EmailPathUtilityFunction = (
+	siteName: string | null | undefined,
+	domainName?: string | null,
+	relativeTo?: string | null,
+	urlParameters?: QueryStringParameters
+) => string;
+
+export const emailManagementPrefix = '/email';
+export const emailManagementAllSitesPrefix = '/email/all';
+
+export function isUnderEmailManagementAll( path?: string | null ) {
```

Comment:
This function isn't used outside of the component, so I would avoid redundant exports, to emphasize that it's just internal helper.
Actually, it's used just for tests https://github.com/Automattic/wp-calypso/pull/86097/files#diff-ebc877d60156092825d6abb9e355604a18e9f61aeee5c48b2383b97ea8ab7eeaR229, and IMO such testing looks not very valuable, I would rather test only the owners (`get...Path`) of this function.

Reply:
Hm, I don't see it 😅 Could you please point me with a link?

Reply:
Here
https://github.com/Automattic/wp-calypso/blob/0eed03aa7e4d0772f2ac0af62500d8128ae43409/client/my-sites/domains/paths.js#L12

Reply:
Thanks, somehow missed it